garage/src/env.ts

```diff
@@ -33,7 +33,7 @@ const mainChainEnvMapping = {
 const secondaryChainEnvMapping = {
   [DeploymentEnvironment.DEVELOPMENT]: chains.polygonMumbai,
   [DeploymentEnvironment.STAGING]: chains.polygonMumbai,
-  [DeploymentEnvironment.PRODUCTION]: chains.filecoin,
+  [DeploymentEnvironment.PRODUCTION]: chains.arbitrum,
```

Comment:
you could use the `tablelandChain` entry here: https://github.com/tablelandnetwork/rigs/blob/main/ethereum/deployments.ts#L50, but nbd

Reply:
Ah true. I'll merge this PR as it is for now, the next time I make any changes here I'll update it to parse it from `tablelandChain` instead